packages/js/examples/react-audio/stories/components/Utilities.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useRef, useState } from 'react';
+import { TelnyxRTC } from '@telnyx/webrtc';
+
+function Utilities({ environment, username, password }) {
+  const clientRef = useRef();
+  const [isConnected, setIsConnected] = useState(false);
+  const [message, setMessage] = useState('');
+
+  const initClient = () => {
+    clientRef.current = new TelnyxRTC({
+      env: environment,
+      login: username,
+      password,
+    });
+  };
+
+  const connect = async () => {
```

Comment:
I initially thought I had to connect to use `getAudioInDevices` but that wasn't the case. I just left it in this PR for future use.